Rules.mk

```diff
@@ -105,12 +105,14 @@ install: cmd/ipfs-install
 install_unsupported:
 	@echo "note: this command has yet to be tested to build in the system you are using"
 	@echo "installing gx"
-	go get -u github.com/whyrusleeping/gx
-	go get -u github.com/whyrusleeping/gx-go
+	go get -v -u github.com/whyrusleeping/gx
+	go get -v -u github.com/whyrusleeping/gx-go
+	@echo check gx and gx-go
+	gx -v && gx-go -v
```

Comment:
I wouldnt remove this. Its rather important to ensure both are installed properly before continuing